code/modules/vehicles/ridden/tug.dm

```diff
@@ -0,0 +1,47 @@
+/obj/vehicle/ridden/tug
+	name = "Train Tug"
+	desc = "A ridable electric car designed for pulling trolleys."
+	icon = 'icons/obj/vehicles/tug.dmi'
+	icon_state = "tug_base"
+	integrity = 200
+	max_integrity = 200
+	riding_handler_type = /datum/component/riding_handler/vehicle/ridden/tug
+
+
+/datum/component/riding_handler/vehicle/ridden/tug
+	vehicle_move_delay = 3
+	allowed_turf_types = list(
+		/turf/simulated,
+		/turf/unsimulated
+	)
+
+/datum/component/riding_handler/vehicle/ridden/tug
```

Comment:
this line shouldnt be needed, you literally define the datum a part above

Reply:
Fair enough, I was simply following how silicons was organizing them (in the boat one. ) IMO it helps keep it from being a bit of a mess to look at, does it harm it at all to do it like this?

Reply:
you can always slap a comment here if you want a visual seperation
I am not sure if it has a noticeable impact, but it might increase compile time
and imo its confusing rather than helping with readability